### ISSUE TITLE
The packagers don't work with the new naming schemes for files in BitBucket

### ISSUE THREAD

**Issue by mhvelplund** [WetDesertRock/lovepacker#2]
From version 11.x and up, mac files no-longer have osname `macosx-64`, just `macos`. Example:

* love-11.2-macos.zip
* love-0.10.2-macosx-x64.zip

Versions after 11.0.0 have version numbers with only two parts in the zip name for windows, but the version in the folder from the zip file has a postfixed '.0'. Example:

```
$ unzip -l ~/.lovepacker/cache/love-11.2-win32.zip
Archive:  /Users/mhv/.lovepacker/cache/love-11.2-win32.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
    76132  11-25-2018 17:57   love-11.2.0-win32/changes.txt
   370070  11-25-2018 17:52   love-11.2.0-win32/game.ico
    66499  11-25-2018 17:57   love-11.2.0-win32/license.txt
  3535872  11-25-2018 18:01   love-11.2.0-win32/love.dll
   385024  11-25-2018 18:01   love-11.2.0-win32/love.exe
   370070  11-25-2018 17:52   love-11.2.0-win32/love.ico
   385024  11-25-2018 18:01   love-11.2.0-win32/lovec.exe
   350720  11-25-2018 17:59   love-11.2.0-win32/lua51.dll
   145408  11-25-2018 17:59   love-11.2.0-win32/mpg123.dll
   455328  10-05-2013 04:38   love-11.2.0-win32/msvcp120.dll
   970912  10-05-2013 04:38   love-11.2.0-win32/msvcr120.dll
   602112  11-25-2018 17:57   love-11.2.0-win32/OpenAL32.dll
     3642  11-25-2018 17:57   love-11.2.0-win32/readme.txt
   863744  11-25-2018 17:58   love-11.2.0-win32/SDL2.dll
---------                     -------
  8580557                     14 files

$ unzip -l ~/.lovepacker/cache/love-11.0.0-win32.zip
Archive:  /Users/mhv/.lovepacker/cache/love-11.0.0-win32.zip
  Length      Date    Time    Name
---------  ---------- -----   ----
    74588  04-01-2018 20:18   love-11.0.0-win32/changes.txt
   370070  04-01-2018 20:17   love-11.0.0-win32/game.ico
    66499  04-01-2018 20:18   love-11.0.0-win32/license.txt
  3531776  04-01-2018 20:21   love-11.0.0-win32/love.dll
   381952  04-01-2018 20:21   love-11.0.0-win32/love.exe
   370070  04-01-2018 20:17   love-11.0.0-win32/love.ico
   381440  04-01-2018 20:21   love-11.0.0-win32/lovec.exe
   350720  04-01-2018 20:19   love-11.0.0-win32/lua51.dll
   145408  04-01-2018 20:19   love-11.0.0-win32/mpg123.dll
   455328  10-05-2013 11:38   love-11.0.0-win32/msvcp120.dll
   970912  10-05-2013 11:38   love-11.0.0-win32/msvcr120.dll
   520704  04-01-2018 20:18   love-11.0.0-win32/OpenAL32.dll
     3642  04-01-2018 20:18   love-11.0.0-win32/readme.txt
   863744  04-01-2018 20:18   love-11.0.0-win32/SDL2.dll
---------                     -------
  8486853                     14 files
```

**Comment by mhvelplund** [comment]
Sorry to keep opening issues ;) I fixed the macosx version in my local version, so i can build for all platforms with 11.0.0, which is ok for me. I'm just posting this for your benefit.

**Comment by WetDesertRock** [comment]
Opening issues is great, it lets me know there is interest in me fixing my
software. This issues was fixed on my local copy for the last jam I did, so
I should still have it.

**Comment by WetDesertRock** [comment]
Fixed in 4a20fd0e21266145cc7eb1a89c5257ee7e965f6b

**Comment by mhvelplund** [comment]
Tested successfully with Love2D 11.2
